csrc/python_frontend/python_bindings.cpp

```diff
@@ -733,6 +733,16 @@ void initNvFuserPythonBindings(PyObject* module) {
                   " was neither symbolic(-1), zero_element(0), broadcast(1), or static(>1).");
             }
 
+            if (contiguity.empty()) {
+              for (const auto dim_size : shape) {
+                if (dim_size == 1) {
+                  contiguity.emplace_back(std::nullopt);
```

Comment:
Interestingly surprised...
I saw the clang-tidy failure earlier and I somehow never associate a `std::nullopt` to call a constructor :face_with_head_bandage: